src/constants/i3s-examples.ts

```diff
@@ -24,6 +24,6 @@ export const EXAMPLES: LayerExample[] = [
   {
     id: "building",
     name: "Building",
```

Comment:
Shall we also change the name of the example? "Turanga Library" probably...

Reply:
Yep, renamed